style.css

```diff
@@ -0,0 +1,68 @@
+body {
+  font-family: Lucida Grande,Lucida Sans Unicode,Lucida Sans,Geneva,Verdana,sans-serif;
+}
+
+.card {
+  width: 30%;
+  border: 1px solid lightblue;
+  display: inline-block;
+  margin: 10px 10px;
+}
+
+.cardtitle {
+  border: 1px dotted lightgrey;
+  text-align: center;
+  margin-top: 0;
+}
+
+.specs,
+.pricing {
+  font-weight: bold;
+}
+
+.productimage {
+  margin: auto 0;
+  text-align: center;
+}
+
+.productimagefile{
+  width: 60%;
+}
+
+.productdescription {
+  margin: 10px;
+}
+
+.availabilityindicator {
+  color: yellow;
+  background-color: darkgrey;
+}
+
+.availabilityindicator span {
+  margin-left: 5px;
+}
+
+.specs,
+.pricing {
+  margin: 12px 0px 10px 10px;
+}
+
+.invalidspecsdate {
+  margin: 0;
+  font-size: 10px;
+  font-weight: normal;
+}
+
+
+.productsize,
+.productweight,
+.pricingdetail1,
+.pricingdetail2,
+.pricingdetail3 {
+  margin-left: 10px;
+}
+
+.specs header h3,
```

Comment:
same down here.  can you make it a little cleaner using a class that you can reuse in both places?